src/test/java/io/neonbee/test/endpoint/odata/ODataReadEntitiesTest.java

```diff
@@ -95,6 +98,24 @@ void existingEntitiesTest(VertxTestContext testContext) {
                 testContext).onComplete(testContext.succeedingThenComplete());
     }
 
+    @Test
+    @Timeout(value = 2, timeUnit = TimeUnit.SECONDS)
+    @DisplayName("Respond with 200 if the service is existing and has test entities including correct inline count")
+    void existingEntitiesWithInlineCountTest(VertxTestContext testContext) {
+        Map<String, String> countQuery = Map.of("$count", "true");
+        Future<HttpResponse<Buffer>> response =
+                requestOData(new ODataRequest(TEST_ENTITY_SET_FQN).setQuery(countQuery));
+        assertOData(response, body -> {
+            JsonObject json = body.toJsonObject();
+            assertThat(json.getMap()).containsAtLeast("@odata.count", 5);
+        }, testContext)
+                .compose(v -> assertODataEntitySetContainsExactly(response,
+                        List.of(EXPECTED_ENTITY_DATA_1, EXPECTED_ENTITY_DATA_2, EXPECTED_ENTITY_DATA_3,
+                                EXPECTED_ENTITY_DATA_4, EXPECTED_ENTITY_DATA_5),
+                        testContext))
```

Comment:
avoid redundant code:
```
 List.of(EXPECTED_ENTITY_DATA_1, EXPECTED_ENTITY_DATA_2, EXPECTED_ENTITY_DATA_3,
                                EXPECTED_ENTITY_DATA_4, EXPECTED_ENTITY_DATA_5)
```
is being used twice in the tests (in this test and in "existingEntitiesTest"). If you declare it as constant you can reuse it and it would make the tests smaller.